lib/tasks/data_migration.rake

```diff
@@ -10,6 +10,22 @@ namespace :data_migration do
     SubscriberListMover.new(**args).call
   end
 
+  desc "Rename an alert type and all combinations of subscriptions to it"
+  task :rename_alert_subscription_lists, %i[from_slug to_slug] => :environment do |_t, args|
+    SubscriberList.where("tags->'alert_type' IS NOT NULL").all.each do |list|
```

Comment:
```suggestion
    SubscriberList.where("tags->'alert_type' IS NOT NULL").find_each do |list|
```

This might be slightly more efficient, although I guess there aren't many of these.